core/icons/scripts/genIcons.ts

```diff
@@ -0,0 +1,31 @@
+// This file generates chainIcons.ts
+// Usage: npm run gen
+
+import svg64 from "svg64";
+import { readFileSync, writeFileSync } from "fs";
+import { chains } from "@wormhole-foundation/sdk-base";
+
+const chainToIcon: { [chain: string]: string } = {};
+
+for (const chain of chains) {
+  const svg = readFileSync(`./src/images/chains/${chain}.svg`, "utf-8");
+  const b64 = svg64(svg);
```

Comment:
I think we can just skip this dep and use `encoding.base64.encode` on the bytes of the SVG file

I think we should also just skip the `data:image/svg+xml;base64,` prefix since it can be added by users directly or set as a const here and some `getter` to prepend it

Reply:
I don't think it's *quite* that simple, but I was considering just doing something similar myself to avoid the dep so I'll go ahead and do that.